hw/top_darjeeling/data/top_darjeeling.hjson

```diff
@@ -61,7 +61,7 @@
     // ref: indicates the clock is used as a reference for measurement.
     srcs: [
       { name: "main", aon: "no",  freq: "1000000000" }
-      { name: "io",   aon: "no",  freq: "1000000000" }
+      { name: "io",   aon: "no",  freq: "250000000" }
```

Comment:
Super nit, could you change 1000000000 to 1_000_000_000. Python allows it and is so much more readable.

Reply:
Let's change that :)